admin/query/src/main/java/org/codice/ditto/replication/admin/query/sites/persist/CreateReplicationSite.java

```diff
@@ -38,6 +38,8 @@
 
```

Comment:
❗️ We should update the DESCRIPTION of this method to be `Creates a replication site. If no rootContext is provided, it will default to "services"`.

Reply:
Changed description